app/views/nextUpdates/NextUpdatesOptOut.scala.html

```diff
@@ -37,8 +37,7 @@
 	nextUpdatesHelper: NextUpdatesHelper
 )
 
-@(currentObligations: NextUpdatesViewModel, optOutViewModel: Option[OptOutViewModel],
-  checks:NextUpdatesQuarterlyReportingContentChecks,
+@(currentObligations: NextUpdatesViewModel, nextUpdatesOptOutViewModel: NextUpdatesOptOutViewModel,
```

Comment:
currentObligations can be named 'obligationsModel' to follow same format as 'optOutModel' below